custom/icds_reports/reports/awc_reports.py

```diff
@@ -242,23 +245,24 @@ def get_awc_reports_pse(config, month, domain, show_test=False):
         date_str = date.strftime("%d/%m/%Y")
         image_data = date_to_image_data.get(date_str)
 
-        if image_data:
-            image_name = image_data['image_name']
-            doc_id = image_data['doc_id']
+        if date > ninety_days_ago:
```

Comment:
would not it automatically be excluded  because `date_to_image_data` wont have any value corresponding to `date_str` key? or this check is needed explicitly h ere?

Reply:
This is needed here. If there is no image data then it creates an object with image link as null
https://github.com/dimagi/commcare-hq/blob/f1876eb0d0847ff8ce2403370909cdae2b887b50/custom/icds_reports/reports/awc_reports.py#L254-L259

Reply:
I see cool